debug_module.tex

```diff
@@ -241,11 +243,11 @@ \section{Program Buffer} \label{programbuffer}
 While these programs are executed, the hart does not leave Debug Mode (see
 Section~\ref{debugmode}).  If an exception is encountered during execution of
 the Program Buffer, no more instructions are executed, the hart remains in Debug
-Mode, and \Fcmderr is set to 3({\tt exception error}).
+Mode, and \Fcmderr is set to 3 ({\tt exception error}).
 
 Executing the Program Buffer may clobber \Rdpc. If that is the case, it must be
 possible to read/write \Rdpc using an abstract command with \Fpostexec not set.
-The debugger must attempt to save \Rdpc between halting and
+The debugger must always save \Rdpc between halting and
```

Comment:
I had imagined a minimal implementation would clobber dpc when executing ebreak from a program buffer/instructions stuffing. Thinking about this more, defining ebreak to not change dpc in Debug Mode is probably simpler and cleaner.